worker/schedule-eid-sync.script.js

```diff
@@ -435,7 +439,7 @@ var service = {
         });
     },
     schedulePatientsWithPendingOrders: function (startDate) {
-        var sql = "replace into  etl.eid_sync_queue(person_uuid) (select distinct uuid from (select t3.uuid, t1.patient_id, t1.order_id, t2.order_id as obs_order_id, t1.date_activated from amrs.orders t1  inner join amrs.person t3 on t3.person_id = t1.patient_id left outer join amrs.obs t2 on t1.order_id = t2.order_id having obs_order_id is null) t where t.date_activated > date('?'))";
+        var sql = "replace into  etl.eid_sync_queue(person_uuid) (select distinct uuid from (select t3.uuid, t1.patient_id, t1.order_id, t2.order_id as obs_order_id, t1.date_activated from amrs.orders t1  inner join amrs.person t3 on t3.person_id = t1.patient_id left outer join amrs.obs t2 on t1.order_id = t2.order_id having obs_order_id is null) t where t.date_activated > date('?') ORDER BY t.date_activated DESC limit 2000)";
         sql = sql.replace('?', startDate);
```

Comment:
The 2000 limit can be taken to a config file to avoid redeployment when one wants to change